### DISCUSSION TITLE
Allow choice of floor color stripes for ride queue tiles.

### DISCUSSION THREAD

**Discussion by jcurbaniak** [OpenRCT2/OpenRCT2#22555] [Feature requests]
Currently, there are only 4 preset stripe colors for ride queue tiles: blue, red, green, and yellow.   Would be nice to have a choice to better match the color scheme of the associated ride.   Would just apply to the two color stripes.  Not looking to tailor the entire tile.

**Comment by ZeeMaji** [comment]
This will require some refactors to the way path data is stored in order to have space for saving color data in path elements but after that it should be fairly trivial to implement a remappable footpath queue surface object.

**Comment by J-PIE-314** [comment]
This suggestion is similar to #20383.

**Comment by J-PIE-314** [comment]
This suggestion was made in part of #15889 and #16035.
